kubectl-pg/cmd/scale.go

```diff
@@ -0,0 +1,77 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	PostgresqlLister "github.com/zalando/postgres-operator/pkg/generated/clientset/versioned/typed/acid.zalan.do/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"log"
+	"strconv"
+)
+
+// scaleCmd represents the scale command
+var scaleCmd = &cobra.Command{
+	Use:   "scale",
+	Short: "Add/remove pods to a Postgres cluster",
+	Long: `Scales the postgres objects using cluster-name.
+Scaling to 0 leads to down time.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		clusterName, _ := cmd.Flags().GetString("clusterName")
+		namespace, _ := cmd.Flags().GetString("namespace")
+		if len(args) > 0 {
+			numberofInstances, _ := strconv.Atoi(args[0])
```

Comment:
it is not a particularly good practice to silently ignore errors. what will happen here if someone enters a non-number in the `numberofInstances` field ?

Reply:
Added the error validation